src/main/scala/cromwell/backend/BackendActor.scala

```diff
@@ -29,6 +37,11 @@ trait BackendActor extends Backend with Actor with ActorLogging {
     case CleanUp => cleanUp()
     case SubscribeToEvent(obj) => subscribeToEvent(obj)
     case UnsubscribeToEvent(obj) => unsubscribeToEvent(obj)
-    case ComputeHash => sender ! computeHash
+    case ComputeHash =>
+      val sndr = sender()
+      computeHash onComplete {
```

Comment:
I did consolidate something similar to this in `reply` in WMA but so far N = 1 here.  :smile: